src/main/java/pl/sdacademy/carrental/repositories/ClientRepository.java

```diff
@@ -0,0 +1,8 @@
+package pl.sdacademy.carrental.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import pl.sdacademy.carrental.domain.Client;
+
+
+public interface ClientRepository extends JpaRepository<Client,Long> {
```

Comment:
Client, spacja Long

Reply:
InteliJ mi tak autoformatuje, muszę to przestawić